CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.1
```

Comment:
Ah, note that `2.1.1` is the last published version - `2.2.0` has not yet been published. We should move this changelog entry down to the 2.2.0 section.

Reply:
Oh yeah, good spot. I've fixed this (and also made the titles a little more consistent in this file).